source/_components/media_player.epson.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: "Epson"
+description: "Instructions on how to integrate Epson projector into Home Assistant."
+date: 2018-03-24 14:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: epson.png
+ha_category: Media Player
+ha_release: 0.68
+ha_iot_class: "Local Push"
+---
+
+The `epson` platform allows you to control a Epson projector from Home Assistant.
+
+To add Epson to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: epson
+    host: 192.168.0.123
+```
+
+{% configuration %}
+host:
+  description: The host name or address of the Epson projector
+  required: true
+  type: string
+port:
+  description: The HTTP port number. Defaults to 80.
+  required: false
+  type: int
+name:
```

Comment:
You can add a key for all the optional items named `default` that has the default value for the item as value.
```yaml
port:
 ...
  default: 80
```